views/books/index.html

```diff
@@ -20,7 +20,7 @@ <h2>Runestone Interactive Library of Books</h2>
         
             {{ for book in book_list: }}
             
-            <div id='list' class='library_entry'>
+            <div style="margin: 25px;" id='list' class='library_entry'>
```

Comment:
Hi,

This would be better if it was in the `runestone-custom-sphinx-bootstrap.css` file.  I know we don't always set a great example, but we are working on getting styling out of html and into css as much as we can.